public/js/controllers/index.js

```diff
@@ -1,24 +1,39 @@
+/* global angular */
 angular.module('mean.system')
-.controller('IndexController', ['$scope', 'Global', '$location', 'socket', 'game', 'AvatarService', function ($scope, Global, $location, socket, game, AvatarService) {
+  .controller('IndexController', ['$scope', 'Global', '$location', 'socket', '$http', '$window', 'game', 'AvatarService', function ($scope, Global, $location, socket, $http, $window, game, AvatarService) {
```

Comment:
Please fix all `hound-ci` complains. You should install ESLint in your editor to help you see these errors before pushing them.